.github/workflows/lint.yml

```diff
@@ -4,28 +4,16 @@
 name: Lint
 
 on:
-#   push:
-    # branches: [ master ]
   pull_request:
     branches: [ master ]
 
 jobs:
   build:
-
     runs-on: ubuntu-latest
-
-    strategy:
-      matrix:
-        node-version: [15.x]
-
     steps:
     - uses: actions/checkout@v2
-    - name: Use Node.js ${{ matrix.node-version }}
```

Comment:
Let's please add back the version to the GHA output even if we don't need technically need `strategy`.  It's valuable when seeing there is an issue to immediately know which Node version is being used.  

It should be easy to fetch from the environment or a GHA variable I'd think?

Reply:
Apparently, it doesn't show it in the PR checks-list
![image](https://user-images.githubusercontent.com/2564094/100927421-b8ccc400-3499-11eb-9e03-6696372fa2d7.png)
and neither in the actions summary
![image](https://user-images.githubusercontent.com/2564094/100927489-cd10c100-3499-11eb-9673-2c002d39338d.png)
but it would be available in the log
![image](https://user-images.githubusercontent.com/2564094/100927516-da2db000-3499-11eb-9bf8-ae44b3c0b3bb.png)

I'm fine reverting to `strategy` if you prefer seeing the version in all those places.

Reply:
What is doing now may be sufficient. This is probably less important for linting than our main CI test.